### ISSUE TITLE
Trevor fails to detect Docker

### ISSUE THREAD

**Issue by MaximDevoir** [vadimdemedes/trevor#84]
I've installed Trevor with `npm install --global trevor`.

When I run Trevor in a repository, I get the following error:

```bash
 $ trevor
 ✖ Docker must be running to run the tests
```

When I run `docker -v`

```bash
 $ docker -v
Docker version 19.03.4, build 9013bf583a
```

The result from [is-docker-running](https://github.com/vadimdemedes/trevor/blob/master/lib/is-docker-running.js) is `true`.

The error is probably coming in `main.js`
https://github.com/vadimdemedes/trevor/blob/4845c9cbb34230c376dbc2fed11ee9fa2d5d3fb1/lib/main.js#L72-L79

**Comment by MaximDevoir** [comment]
Maybe #81 fixes this.

NPM has an old version from this repository.

@vadimdemedes Can you update the package on NPM?

**Comment by MaximDevoir** [comment]
The reason Trevor is failing is because `docker version` may require root permissions. `docker -v` does not require root permissions.

I was able to get the program to work, but I had to run Node and Trevor as root (probably not a good idea).

**Comment by vadimdemedes** [comment]
Done, published `2.4.1`. Let me know if it fixes the issue.

**Comment by MaximDevoir** [comment]
I won't be able to test the latest published package. But, the published version should fix the issue. If I remember correctly, I ran a local version of Trevor with #81 merged and the error went away.
